### PR TITLE
Sad path

### LINKED ISSUE #15: Sad Path/ Edge Case Testing
- [x] Implement sad path/edge case testing for request specs

### PR DESCRIPTION
**Pull request by jmichlitsch**
### What Changed?
- Base Sad Path Testing for the microservice.
### What's Next?
- Make more robust error handling to give more error possibilities 
### Known Issues?
- No
### Test Coverage?
- 100%
### README & Documentation
- _Think: Does this PR add something that should be documented? If not, you can remove this section._
- What did you add to the README or project documentation?

### CODE REVIEW

File: spec/requests/distance_request_spec.rb [hunk 1]
```diff
@@ -5,24 +5,62 @@
...
+      error = JSON.parse(last_response.body, symbolize_names: true)
 
-      expect(last_response).to be_ok
+      expect(error).to be_a(String)
+      expect(error).to eq('Parameter(s) missing: Please provide a valid origin and destination')
+    end
+
+    it 'returns errors for invalid location' do
+      params = {
+        origin: 'doifhdsfhdjsfl',
```
Comment (klatour324) [inline spec/requests/distance_request_spec.rb:46]: This is great sad path coverage! As I mentioned to Alexa, and this was just coming from my experience with my Sweater Weather eval.. a great edge case we can test for would be if the origin and destination are something like `[]` or `&&&&&` or some real odd characters outsides of just letters. But this is definitely a great start to the sad path testing!

**Review by klatour324 — APPROVED** [review]
I am so happy you were able to take time to get these error statuses handled correctly with Sinatra! Just a small note that I had about edge case testing that we can look forward to adding once we complete our BE-Microservice tickets finished. Great work!
